README.md

```diff
@@ -1,2 +1,17 @@
 # dashboard-hkdistrictcouncillors
 Repository for code underlying the Shiny app dashboard for HK district councillors
+
+## Background
+The aim of this app is to provide a live one-stop-shop for information on the district councillors in Hong Kong, via aggregating posts and feeds from their public Facebook pages. 
```

Comment:
On the phrasing *"one-stop-shop"*, does it make sense to non-native English speakers? Might be better to replace it with something that's less of an English idiom, such as *"single, convenient source of information..."*.

Reply:
I think it's not an issue because there's an actual direct translation in it in Chinese ("一站式") that would make sense to people - but at the same not wedded to this, happy to make it clearer. Over time I think it would be good to have a Chinese version for the README, but we can definitely worry about that later!

Reply:
Ah, no problem, I never knew. A Chinese version would be good though I think Ocean will have to review because I have no hope on it!